website/vault.erb

```diff
@@ -421,6 +429,18 @@
                             <a href="/docs/providers/vault/r/ldap_auth_backend_group.html">vault_ldap_auth_backend_group</a>
                         </li>
 
+                        <li<%= sidebar_current("docs-vault-resource-ldap-secret-backend") %>>
+                            <a href="/docs/providers/vault/r/ldap-secret-backend.html">vault_ldap_secret_backend</a>
+                        </li>
+
+                        <li<%= sidebar_current("docs-vault-resource-ldap-secret-backend-static-role") %>>
```

Comment:
Looks like we're missing the dynamic role resource from the sidebar?

Reply:
thanks!